internal/clusterfeature/features/logging/operator.go

```diff
@@ -506,3 +439,52 @@ func mergeValuesWithConfig(chartValues interface{}, configValues interface{}) ([
 
 	return json.Marshal(result)
 }
+
+func (op FeatureOperator) createLoggingResource(ctx context.Context, clusterID uint, spec featureSpec) error {
+	// TODO (colin): add labels
+	var tlsEnabled = spec.Logging.TLS
+	var loggingResource = &v1beta1.Logging{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      loggingResourceName,
+			Namespace: op.config.Namespace,
+		},
+		Spec: v1beta1.LoggingSpec{
+			FluentbitSpec: &v1beta1.FluentbitSpec{
+				Image: v1beta1.ImageSpec{
+					Repository: op.config.Images.Fluentbit.Repository,
+					Tag:        op.config.Images.Fluentbit.Tag,
+					PullPolicy: "IfNotPresent",
+				},
+				TLS: v1beta1.FluentbitTLS{
+					Enabled: tlsEnabled,
+				},
+				Metrics: &v1beta1.Metrics{
+					ServiceMonitor: spec.Logging.Metrics,
+				},
+			},
+			FluentdSpec: &v1beta1.FluentdSpec{
+				TLS: v1beta1.FluentdTLS{
+					Enabled: tlsEnabled,
+				},
+				Image: v1beta1.ImageSpec{
+					Repository: op.config.Images.Fluentd.Repository,
+					Tag:        op.config.Images.Fluentd.Tag,
+					PullPolicy: "IfNotPresent",
+				},
+				Metrics: &v1beta1.Metrics{
+					ServiceMonitor: spec.Logging.Metrics,
+				},
+			},
+			ControlNamespace: op.config.Namespace,
+		},
+	}
+
+	if tlsEnabled {
+		loggingResource.Spec.FluentdSpec.TLS.SecretName = fluentSharedSecretName
+		loggingResource.Spec.FluentdSpec.TLS.SharedKey = fluentSharedSecretName
```

Comment:
I beleive this should be a random generated string instead of the name of a secret